gnosis/cowswap
# [Bugfix] Move About item from Header to ellipsis menu

# Summary

Fixes #995 

*As part of the issue tagged above, `About` item was moved from Header component to the ellipsis menu*

![image](https://user-images.githubusercontent.com/11525018/127550114-7a5c9075-c481-49af-910c-1186703c64d7.png)

![image](https://user-images.githubusercontent.com/11525018/127550267-225912c2-ad6b-439c-8ff2-fb6f5cc8e7b6.png)



  # To Test

1. Open `swap` or `profile`
- [x] Verify the header component does not contains the `about` page
2. Click on the ellipsis in the top right corner
- [x] Verify the menu contains `About` page
- [x] Check if clicking on this item goes to `about` page
